tsconfig.base.json

```diff
@@ -290,6 +290,9 @@
       "@energinet-datahub/gf/util-browser": [
         "libs/gf/util-browser/src/index.ts"
       ],
+      "@energinet-datahub/watt/HTMLElementEvent": [
+        "libs/ui-watt/src/lib/utils/HTMLElementEvent.ts"
+      ],
```

Comment:
issue: the library doesn't exist anymore. Can be removed. I think it's this that triggers EO's checks.

Reply:
A forgot to remove that